google-cloud-bigquery/src/main/scala/akka/stream/alpakka/googlecloud/bigquery/impl/auth/ServiceAccountCredentials.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.googlecloud.bigquery.impl.auth
```

Comment:
We agreed multiple times, that this package should be a separate project/module. Probably if this get merged, I will sbt-hack it to use this in every google related project and stop duplicating code... (But I like the way as it gets bigger and bigger :D)

Reply:
Yes, I saw this mentioned many times, the main discussion is in #773? I would fully support this!

Reply:
The consensus in #773 seems to be to factor out Google auth things into an external library. If either of you are interested in starting a project we can bring it under the akka organization (perhaps `akka/alpakka-google-auth`) and setup a release for it.

Realistically this is probably something the akka team will get to soon, but I'm willing to help facilitate the integration/library/release/etc.

Reply:
I'm happy to help! Actually, I started to play with it at some point, but the main question is "where?" remained unclear. I think it would be convenient to do it here, but also this project starting to be huge. If I get a green lamp, I'm happy to start to refactor the common google part to a subproject.